haystack/nodes/retriever/dense.py

```diff
@@ -1507,6 +1508,7 @@ def __init__(
                                   If no value is provided, a default empty list will be created.
         :param api_key: The OpenAI API key or the Cohere API key. Required if one wants to use OpenAI/Cohere embeddings.
                         For more details see https://beta.openai.com/account/api-keys and https://dashboard.cohere.ai/api-keys
+        :param api_base: The OpenAI API base URL, defaults to `"https://api.openai.com/v1"`
```

Comment:
Let's move this to the end of doc strings given that we'll move `api_base` to the end of the arguments.